src/workerd/api/system-streams.c++

```diff
@@ -299,7 +299,7 @@ kj::Promise<void> EncodedAsyncOutputStream::end() {
   return promise.attach(ioContext.registerPendingEvent());
 }
 
-void EncodedAsyncOutputStream::abort(kj::Exception reason) {
+void EncodedAsyncOutputStream::abort(kj::Exception reason, AbortOption ignored) {
   inner.init<Ended>();
```

Comment:
Interesting to note that for compressed streams, this implementation has always failed to flush the buffer before aborting, so the compat flag doesn't make a whole lot of difference (either way, previously-written data may be lost).

Makes me wonder if the compat flag is even needed.

Reply:
Yeah that's a good point... hmmm

Reply:
I think the compat flag is still likely necessary. There's an observable difference in behavior around the rejected queued write promises here that someone could be currently working around ( at least in theory )...likely safest to have the flag

Reply:
but I think you're right about not needing the `AbortOption` enum class now.